Samples/Notifications/Push/cpp-console-unpackaged/cpp-console-unpackaged.cpp

```diff
@@ -92,10 +92,22 @@ void SubscribeForegroundEventHandler()
 
 int main()
 {
-    // Setup an event handler, so we can receive notifications in the foreground while the app is running.
-    SubscribeForegroundEventHandler();
+    auto pushNotificationManager{ PushNotificationManager::Default() };
 
-    PushNotificationManager::Default().Register();
+    if (pushNotificationManager.IsSupported())
+    {
+        // Setup an event handler, so we can receive notifications in the foreground while the app is running.
+        SubscribeForegroundEventHandler();
+
+        pushNotificationManager.Register();
+    }
+    else
+    {
+        // Here, the app should handle the case where push notifications are not supported, for example:
+        // - maintian its own persistent connection with an App Service or
```

Comment:
typo: maintain

Reply:
dyslexic fingers... thanks for catching this